### DISCUSSION TITLE
HEIC conversion issue

### DISCUSSION THREAD

**Discussion by okarpov** [dlemstra/Magick.NET#767] [Help]
Opening HEIC file and converting it into Bitmap seems buggy

### Description
Bitmap is distorted
![Screenshot_8](https://user-images.githubusercontent.com/2629775/97779577-9b999280-1b87-11eb-8fb3-606dab17322c.png)

[IMG_0050.zip](https://github.com/dlemstra/Magick.NET/files/5469080/IMG_0050.zip)

### Steps to Reproduce
                ```
                FileOpenPicker ofp = new FileOpenPicker();
                ofp.FileTypeFilter.Add(".heic");
                var ofr = await ofp.PickSingleFileAsync();

               Stream ss = ofr.OpenStreamForReadAsync().Result;


                var image = new MagickImage(ss);
                image.Format = MagickFormat.Bmp3;
                WriteableBitmap res = new WriteableBitmap(imageMagick.Width, imageMagick.Height);
                
                var data = image.ToByteArray();
                res.PixelBuffer.AsStream().Write(data, 0, data.Length);
                ```

### System Configuration

- Magick.NET version: Magick.NET-Q8-x64 7.22.0
- Environment (Operating system, version and so on): Windows 10
- Additional information: UWP

**Comment by dlemstra** [comment]
You might want to consider using this packge instead: https://www.nuget.org/packages/Magick.NET.SystemWindowsMedia/. It will allow you to do this:

```C#
var res = image.ToBitmapSource();
```
